src/main/java/trivia/Board.java

```diff
@@ -5,20 +5,20 @@
 
 class Board {
 
-  private String[] positionToQuestionCategoryName;
+  private final QuestionCategoryName[] positionToCategory;
```

Comment:
Instead of having QuestionCategoryName and QuestionCategory, this can be QuestionCategory and QuestionDeck.

positionToCategory is just **categories**.

Name doesn't have to be connected with the way it will be used later, like here it is using word position to imply that it will be read by position.

Reply:
This field represents a mapping from a position on the board to the associated question category. In general I prefer to name these kinds of fields in a way that explicitly reveals the intent behind the mapping scheme.

Reply:
In that case I can see orderedCategories or positionedCategories as another suggestion